cmd/installer/cmd/root.go

```diff
@@ -23,9 +25,34 @@ var rootCmd = &cobra.Command{
 	Long:  ``,
 }
 
+const metaValueEnvVariable = "INSTALLER_META_VALUES_BASE64"
+
+func parseEnvironment() error {
+	if metaEnvBase64 := os.Getenv(metaValueEnvVariable); metaEnvBase64 != "" {
+		metaEnv, err := base64.StdEncoding.DecodeString(metaEnvBase64)
+		if err != nil {
+			return err
+		}
+
+		for _, val := range strings.Split(string(metaEnv), ";") {
+			if err := options.MetaValues.Set(val); err != nil {
+				return err
+			}
+		}
+	}
+
+	return nil
+}
+
 // Execute adds all child commands to the root command and sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
+	// Set defaults for flags from the environment variables.
+	if err := parseEnvironment(); err != nil {
```

Comment:
rename to setEnvironment, since it also sets meta?

Reply:
I mean it parses environment variables and sets flags (defaults) based on env variables, but probably name is not good one